include/swift/AST/GenericSignatureBuilder.h

```diff
@@ -1546,17 +1546,16 @@ class GenericSignatureBuilder::PotentialArchetype {
   /// path compression on the way.
   PotentialArchetype *getRepresentative() const;
 
+private:
   /// Retrieve the generic signature builder with which this archetype is
   /// associated.
-  /// FIXME: Only EquivalenceClassVizIterator gets to use this.
   GenericSignatureBuilder *getBuilder() const {
```

Comment:
So is this still needed?

Reply:
There are several callers still, but not so many that we can't eliminate their dependence on `getBuilder()`.

Reply:
*sigh*, that took longer than expected. It's gone now.